nomad/node_pool_endpoint.go

```diff
@@ -134,6 +137,9 @@ func (n *NodePool) GetNodePool(args *structs.NodePoolSpecificRequest, reply *str
 		return structs.ErrPermissionDenied
 	}
 
+	// Only warn for expiration of a read request.
+	_ = n.validateLicense(nil)
```

Comment:
This only warns in the logs, right? The user running `nomad node pool list` doesn't see anything? (I guess we have no way of reporting warnings in the typical RPCs though, so :shrug: )

Reply:
Yeah, it's a bit weird in this context, but in Enterprise code there's a flag to emit logs that triggers this block:
https://github.com/hashicorp/nomad-enterprise/blob/main/nomad/license_watcher_ent.go#L173-L183